### ISSUE TITLE
3.0 release

### ISSUE THREAD

**Issue by kleuter** [processhacker/processhacker#415]
Will [3.0](https://github.com/processhacker/processhacker/releases) ever release officially? It's been already 3 years since 2.39, just curious.
Thanks, this amazing tool is the first I install on any PC.

**Comment by krokofant** [comment]
Related
#413 
> There was a release four days ago? 😃
> https://wj32.org/processhacker/nightly.php

#360 
#255
> There hasn't been a compelling reason to update the stable release yet... I will update the site when a stable release is ready.

**Comment by kleuter** [comment]
Got it, it's probably a policy of @dmex not to release any more.

**Comment by krokofant** [comment]
@kleuter Seems so, but I would reopen the issue since we haven't really got a clear answer.

**Comment by AltF5** [comment]
Is there reason to believe 3.0 is more unstable than 2.39? Perhaps other things need taking care of like driver signing?

**Comment by kleuter** [comment]
Nope, the driver in both 2.39 & 3.0 is the same, signed on 9 April 2016

**Comment by Bodup** [comment]
I'm still using 2.39 as well but I've noticed a lot of new useful features in the current nightlies so I'd love to see a new (signed) stable release as well.

**Comment by Tharn** [comment]
Nightlies are hit or miss. Some work great, but a lot of recent builds I tried have produced crashes.

**Comment by AltF5** [comment]
The only crashes I’ve seen are with User CPU Time and Kernel CPU Time. 

What’s other crashes have you noticed?

**Comment by Tharn** [comment]
Sometimes when starting and navigating around PH.
Just today nightly v. 2471 crashed when clicking 'exit'.
I looked around but can't find a crash log unfortunately.

**Comment by Azumgi** [comment]
A new release would be very convenient for [Scoop](https://github.com/lukesampson/scoop-extras/blob/master/bucket/processhacker.json) and [Chocolatey](https://chocolatey.org/packages/processhacker) users.

Perhaps, there can be some roadmap to see what has to be done before 3.0 is ready?

**Comment by rugk** [comment]
> Perhaps, there can be some roadmap to see what has to be done before 3.0 is ready?

You can use GitHub milestones (or alternatively projects) for that.
